### DISCUSSION TITLE
Welcome to developer-portal Discussions!

### DISCUSSION THREAD

**Discussion by Mercurial** [cardano-foundation/developer-portal#158] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ALVIN-OBONYO** [comment]
Hello, my name is Alvin Obonyo a student from Nairobi, Kenya. I am glad to be here and I would love to get my feet on how to be a Cardano developer and contribute to it.
